orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/temporal/AfterOrAtField.feature

```diff
@@ -0,0 +1,26 @@
+Feature: User can specify that one date should be after or equal to another date
+
+  Background:
+    Given the generation strategy is full
+
+  Scenario: Running an "afterOrAtField" constraint allows one date to be always later than or equal to another
+    Given there is a field foo
+    And foo is of type "datetime"
+    And foo is after 2018-09-01T00:00:00.000Z
```

Comment:
Take this out and it should make the first one equal

Reply:
You are not testing the "OrAt" bit of the statement

Reply:
To test it, you could change the above line to
`And bar is after 2018-09-01T00:00:00.000Z`